packages/serverpod/lib/src/server/message_central.dart

```diff
@@ -69,19 +69,23 @@ class MessageCentral {
       subscribedChannels = {};
       _sessionToChannelNamesLookup[session] = subscribedChannels;
     }
-    subscribedChannels.add(channelName);
 
-    var callbacks = _sessionToCallbacksLookup[session];
-    if (callbacks == null) {
-      callbacks = {};
-      _sessionToCallbacksLookup[session] = callbacks;
-    }
-    callbacks.add(listener);
+    // Only suscribe if it's not already done
+    if (!subscribedChannels.contains(channelName)) {
```

Comment:
This part made sense when developing locally, but in practice it might not…

There seems to be a valid use-case where one wants to subscribe multiple handlers e.g. if a user has multiple web-sockets open etc.